contents/handbook/strategy/strategy.md

```diff
@@ -5,41 +5,85 @@ showTitle: true
+
+
+### How should we prioritize between competing directions?
+
+While there is value in the items on the right, we value the items on the left more.
+
+**Breadth vs. depth of capabilities**
+
+* _Breadth_: build basic versions of every feature needed rather than a small set of sophisticated ones, so our customers can consolidate.
+
+**High quality users vs. high paying businesses**
+
+* _High quality users_: focus on acquiring more high quality users over big ticket contracts, so we can get better feedback and learn faster.
+
+**Plugin-based platform vs one-stop-shop**
+
+* _Plugin-based platform_: to move faster, look for ways to integrate with the best tools to solve customer problems over building every feature ourselves.
```

Comment:
It feels like this guideline, as written, contradicts with the 'Breadth vs depth' guideline.

I might be reading into the 'Breadth vs depth' one too much, but I was interpreting it as "let's add more features to PostHog, so that we're the only self-hosted product our customers need", and I'm reading the 'plugin-based platform' one as "lets integrate with 3rd parties, so we don't need to build the features ourselves"

I'm wondering if the 'Plugin-based platform' one is more about integrating 3rd parties as data sources? If that's the case, maybe we need to change the 'over building every feature ourselves' part?

Reply:
```suggestion
* _Plugin-based platform_: integrate with the best services and data sources to solve customer problems faster
```
I agree - it needs a bit of tweaking to be clearer as they shouldn't contradict, does this suggestion feel right?